lib/main.js

```diff
@@ -55,6 +55,11 @@ function bind(context, fn) {
   };
 }
 
+// Define textContent method to safely fetch textContent of unknown elements
+function textContent(elem, defaultText) {
```

Comment:
Can we rename this to `getTextContent`? It will feel more semantic to me; right now I am not sure whether this would be a get/set/something else